src/components/caption/Caption.vue

```diff
@@ -215,7 +215,7 @@ export default {
         }
 
         .vs-caption__caption-info{
-            padding: $spacer-100;
+            padding: $spacer-100 0;
```

Comment:
Should this be `$spacer-0` instead of `0`?

Reply:
Just for consistency really, I noticed a PR the other day where 0s were changed to $spacer-0 but it's not a massive issue